Classes/Backend/TceMain.php

```diff
@@ -786,6 +802,15 @@ protected function getClipboardCommand()
     }
 
     /**
+     * @return array
+     */
+    protected function getCopyPasteCommand()
+    {
+        $command = GeneralUtility::_GET('cmd');
```

Comment:
I'm still not convinced we need to read this command. In the method above where this method gets used, the existing case is there to catch a very specific operation: triggering when you do clipboard-paste commands. It must not trigger after drag-n-drop commands (the moveRecord_* methods should take care of that; they must trigger after drag-n-drop ends).

What I'm thinking is that maybe the moveRecord_* method to move-to-top-of-column is doing something wrong or missing some logic? In any case: those are the methods expected to handle this type of command!